ui/src/config/section/compute.js

```diff
@@ -32,9 +32,9 @@ export default {
       permission: ['listVirtualMachinesMetrics'],
       resourceType: 'UserVm',
       params: () => {
-        var params = {}
+        var params = { details: 'servoff,tmpl,nics' }
         if (store.getters.metrics) {
-          params = { state: 'running' }
+          params = { details: 'servoff,tmpl,nics,stats' }
```

Comment:
VM stats in the API response can cause 5-10x delay compared to normal list API response

Reply:
is `tmpl` needed in the details? or `stats`?

Reply:
my concern is opposite to @DaanHoogland  :-D
is `'servoff,tmpl,nics`' or `servoff,tmpl,nics,stats` enough for UI ?

Reply:
Yes, these are enough just for list view (table view); the stats are only needed when the metrics button is clicked. For the resource view, we delete the params.details, so by default details=all is used when listing a single VM.

Reply:
@weizhouapache @DaanHoogland 
I tested this again both as root admin and normal user account; for VMs and templates with and without resource icons; these are enough details to render the list view (table); when the user clicks any VM/instance, then we call the API remove the details and all details are listed/used in the resource view. While not ideal, this speeds up the listing.

Reply:
great, thanks @rohityadavcloud 
I will test it